README.md

```diff
@@ -255,6 +256,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [Catalyst](https://github.com/PennyLaneAI/catalyst) - AOT/JIT compiler for hybrid quantum computing beyond NISQ. Written in MLIR. Python frontend via [PennyLane](https://pennylane.ai).
 - [QCOR](https://github.com/ORNL-QCI/qcor) - C++ language extension and associated compiler implementation for hybrid quantum-classical programming.
 - [ScaffCC](https://github.com/epiqc/ScaffCC) - Compilation, analysis and optimization framework for the Scaffold quantum programming language.
+- [TKET](https://github.com/CQCL/tket) - C++ library for placement, routing and optimization of quantum circuits (developed by Quantinuum).
```

Comment:
Should [TKET v2](https://github.com/CQCL/tket2) be added under "Rust" as well?

Reply:
Good point, added